pkg/3scale/amp/component/apicast_options_builder.go

```diff
@@ -9,12 +9,14 @@ import (
 
 type ApicastOptions struct {
 	// required options
-	appLabel       string
-	managementAPI  string
-	openSSLVerify  string
-	responseCodes  string
-	tenantName     string
-	wildcardDomain string
+	appLabel           string
+	managementAPI      string
+	openSSLVerify      string
+	responseCodes      string
+	tenantName         string
+	wildcardDomain     string
+	productionImageTag string
+	stagingImageTag    string
```

Comment:
Decided to provide the tag value as an option of the component, having one option per DeploymentConfig (configurable at DeploymentConfig level)

Reply:
:+1: